### DISCUSSION TITLE
Version 1.0 Documentation

### DISCUSSION THREAD

**Discussion by ovu** [arrow-kt/arrow#2526] [General]
Hi all,

I just read in twitter that the version 1 is stable is released. Do you have any Documention about what is new in Version 1?

Thanks and keep the good work!

-ovu

**Comment by nomisRev** [comment]
Hey @ovu,

Yes the release post was published on the 47 Degrees website [here](https://www.47deg.com/blog/arrow-1-0-release/)

Perhaps also interessting is the release post of the last two version since we've been working towards 1.0 since we've stabilised the API and features in those versions rather than in 1.0 itself.

- [`0.12.x/0.13.x`](https://www.47deg.com/blog/arrow-0.13.0-release/) these are the 2 version before 1.0.0 that converged to the 1.0.0 API

**Comment by ovu** [comment]
@nomisRev Thanks for the information!
